articles/fleet-4.17.0.md

```diff
@@ -0,0 +1,88 @@
+# Fleet 4.17.0 | Better osquery management, user engagement, improved host vitals.
+
+![Fleet 4.17.0](../website/assets/images/articles/4.17.0-cover-1600x900@2x.jpg)
+
+Fleet 4.17.0 is up and running. Check out the full [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.17.0) or continue reading to get the highlights.
+
+For update instructions, see our [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.
+
+## Highlights
+- Manage osquery across your organization. 
+- Engage with your users through Fleet Desktop.
+- Battery condition added to **Host Details** for MacOS.
+
+## Manage osquery across your organization.
+**Available in Fleet Free and Fleet Premium**
+
+![Manage osquery across your organization..](../website/assets/images/articles/fleet-4.17.0-1-1600x900@2x.jpg)
+Fleet's osquery installer is out of beta! Orbit is a lightweight wrapper for osquery that automates the process of installing, updating and configuring osquery on all of your devices. Even if you're not using Fleet!
+
+Future plans for the installer include adding the ability to manage `osquery` extensions and enrollment flags across your organization without needing to reinstall anything or ssh in to individual machines. A simple, one-stop shop for remotely managing `osquery`.
+
+You can generate your installer using [`fleetctl package`](https://fleetdm.com/docs/deploying/fleetctl-agent-updates#packaging-with-orbit). If you're enrolling your host to a Fleet server, he best place to find the full command with flags specific to your Fleet instance is **Hosts > Add hosts** in the Fleet UI. Once the installer has been generated, just extract the file and run it on the host!
```

Comment:
```suggestion
You can generate your installer using the [`fleetctl`] CLI tool. (https://fleetdm.com/docs/deploying/fleetctl-agent-updates#packaging-with-orbit). If you're enrolling your host to a Fleet server, he best place to find the full command with flags specific to your Fleet instance is **Hosts > Add hosts** in the Fleet UI. Once the installer has been generated, just extract the file and run it on the host!
```

Suggested change to specify that fleetctl is a command line tool. This is because the user users fleetctl to generate the package. They run the `fleetctl package` command. I really could be overthinking this. Up to you